app/utils.js

```diff
@@ -500,6 +500,44 @@ function refreshExchangeRates() {
 	}
 }
 
+function getExchangeRatesUpdateTime() {
+	return global.exchangeRatesUpdateTime;
+}
+
+function getExchangeFromExchangeRateExtensions() {
+	// Any other kind of currency conversion must use the BCHUSD base conversion rate to be calculated, in consecuence --no-rates must be disabled.
+	if (coins[config.coin].exchangedRateDataExtension.length > 0 && global.exchangeRates['usd']) {
+		coins[config.coin].exchangedRateDataExtension.forEach(exchangeProvider => {
+			request(exchangeProvider.jsonUrl, function(error, response, body) {
+				if (error == null && response && response.statusCode && response.statusCode == 200) {
+					var responseBody = JSON.parse(body);
+	
+					var exchangeRates = exchangeProvider.responseBodySelectorFunction(responseBody);
+					if (exchangeRates != null || Object.entries(exchangeRates).length > 0) {
+						var originalExchangeRates = global.exchangeRates;
+						var extendedExchangeRates =  {};
+						for	(const  key in exchangeRates) {
+							extendedExchangeRates[key] = (parseFloat(originalExchangeRates.usd) * parseFloat(exchangeRates[key])).toString();
+						}
+						global.exchangeRates = {
+							...originalExchangeRates,
+							...extendedExchangeRates
+						}
+						global.exchangeRatesUpdateTime = new Date();
```

Comment:
since this date is displayed every where with `BTCEXP_NO_RATES=false` (which is the default behavior) I think we need to find a more compact representation otherwise we risk to clutter the UI of the app 

Currently the default format is something like:

`Mon Oct 18 2021 18:13:08 GMT+0200 (Central European Summer Time)`

 Guess that we could use something shorter, e.g.
 
 `2021/10/18 18:13 CEST` 
 
 We could also think a way to reduce the amount of time we actually display such value, e.g. on the main page which contains a partial which renders the block list, for every block we list the same date and time 3 times. 
 
 
![Screenshot 2021-10-18 at 18 17 43](https://user-images.githubusercontent.com/1469203/137769828-d064cb39-7481-4c56-92df-327b6ad692be.png)

Reply:
We decided to remove the changes related to adding the local time related to the latest exchange rate refresh, we will send it as a separate PR.